src/main/webapp/web.xml

```diff
@@ -3,9 +3,10 @@
          xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee
                       http://xmlns.jcp.org/xml/ns/javaee/web-app_4_0.xsd"
          version="4.0">
-    <servlet>
+   <servlet>
         <servlet-name>index</servlet-name>
         <servlet-class>mate.controller.IndexController</servlet-class>
+        <load-on-startup>1</load-on-startup>
     </servlet>
```

Comment:
Why did you add load-on-startup, for practice or does it make sense? The controller is not too big and it is easy to load